axiom_core.c

```diff
@@ -538,6 +560,17 @@ struct input_dev *axiom_register_input_subsystem(void)
 	// Declare that we support "RAW" Miscellaneous events
 	set_bit(MSC_RAW, input_dev->mscbit);
 #endif
+	if (poll_enable) {
+		ret = input_setup_polling(input_dev, axiom_poll);
+		if (ret) {
+			pr_err("Unable to set up polling: %d\n", ret);
+			return NULL;
+		}
+		if (poll_interval)
+			input_set_poll_interval(input_dev, poll_interval);
+		else
+			input_set_poll_interval(input_dev, POLL_INTERVAL_DEFAULT);
```

Comment:
@kamel-bouhara This reference to `POLL_INTERVAL_DEFAULT` needs to be updated to `POLL_INTERVAL_DEFAULT_MS`.

Reply:
Sorry, this is really fixed now !